### PR TITLE
docs: add badges

### PR DESCRIPTION
**Pull request by seriouszyx**
Signed-off-by: “seriouszyx” <seriouszyx@foxmail.com>

Fix: https://github.com/casdoor/casdoor-java-sdk/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # casdoor-java-sdk
 
+[![GitHub Actions](https://github.com/casdoor/casdoor-java-sdk/actions/workflows/maven-ci.yml/badge.svg)](https://github.com/casdoor/casdoor-java-sdk/actions/workflows/maven-ci.yml)
+[![codebeat badge](https://codebeat.co/badges/44f7a8fb-1eca-4590-a1f0-ae13a053857e)](https://codebeat.co/projects/github-com-seriouszyx-casdoor-java-sdk-master)
```
Comment (hsluoyz) [inline README.md:4]: should be our org here?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+[![codecov](https://codecov.io/gh/seriouszyx/casdoor-java-sdk/branch/master/graph/badge.svg?token=5DVSWSOGLQ)](https://codecov.io/gh/seriouszyx/casdoor-java-sdk)
```
Comment (hsluoyz) [inline README.md:5]: should be our org here?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by seriouszyx** [comment]
@hsluoyz Codebeat and Codecov access request for Casdoor Organization has been sent, plz approve.

**Comment by hsluoyz** [comment]
@seriouszyx approved.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**